src/main/java/org/jenkinsci/plugins/gitclient/CliGitAPIImpl.java

```diff
@@ -23,7 +23,16 @@
 import hudson.plugins.git.Revision;
 import hudson.util.ArgumentListBuilder;
 import hudson.util.Secret;
-import java.io.*;
+import java.io.BufferedReader;
```

Comment:
This must have happened because of my IDE settings to never import `*`. 

Should I undo this?

Reply:
No, I like the change.  The style guide for the plugin says that wild card imports should only be used in very narrow cases for tests.